opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -407,10 +449,12 @@ namespace Opm
                         perf_vap_wat_rate = getValue(rvw) * (getValue(cq_s[gasCompIdx]) - getValue(rs) * getValue(cq_s[oilCompIdx])) / d;
                     }
                 }
-                else if (FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx) && FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)) {
+                if (FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx) && FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)) {
```

Comment:
removing `else` here make it not exclude the oil phase anymore. Will it affect the correctness of the code? @totto82

Reply:
I think @atgeirr and I agree that this change was correct even though it affect the results. Do you think otherwise?